gitlab/table_issue.go

```diff
@@ -47,6 +55,34 @@ func listIssues(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData)
 	return listAllIssues(ctx, d, h)
 }
 
+func getIssue(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Debug("getIssue", "started")
+	conn, err := connect(ctx, d)
+	if err != nil {
+		plugin.Logger(ctx).Error("getIssue", "unable to establish a connection", err)
+		return nil, fmt.Errorf("unable to establish a connection: %v", err)
+	}
+
+	q := d.EqualsQuals
+	if q["id"] == nil {
+		return nil, nil
+	}
+
+	id := int(q["id"].GetInt64Value())
+
+	plugin.Logger(ctx).Debug("getIssue", "id", id)
+	issue, _, err := conn.Issues.GetIssueByID(id)
+	if err != nil {
+		plugin.Logger(ctx).Error("getIssue", "id", id)
+		return nil, fmt.Errorf("unable to obtain issue for issue id %d\n%v", id, err)
+	}
+
+	d.StreamListItem(ctx, issue)
+
+	plugin.Logger(ctx).Debug("getIssue", "completed successfully")
+	return nil, nil
+}
+
```

Comment:
As discussed in PR comments, this actually breaks things for non-Admin users: [see here](https://docs.gitlab.com/ee/api/issues.html#single-issue)

Reply:
Yep.  I'll revert that commit.

Reply:
Reverted.  But I fixed `target_iid` so might add the `GetIssue` optimisation.